sdk/lib/rtl/network.c

```diff
@@ -515,7 +516,11 @@ RtlIpv6AddressToStringA(
     ASSERT(Result >= Buffer);
     ASSERT(Result < Buffer + RTL_NUMBER_OF(Buffer));
 
-    Status = RtlUnicodeToMultiByteN(S, RTLIPV6A2S_MAX_LEN, NULL, Buffer, (wcslen(Buffer) + 1) * sizeof(WCHAR));
+    Length = wcslen(Buffer);
+    if (Length > ((ULONG_MAX / sizeof(WCHAR)) - 1))
+        return (PSTR)~0;
```

Comment:
I am quite surprised that this function returns not NULL pointer, but instead "~0" . Is that behaviour tested in our winetests/apitests?